### PR TITLE
Sql bolt task

### PR DESCRIPTION
**Pull request by Hexoplon**

### CODE REVIEW

File: templates/cockroachdb.service.epp [hunk 1]
```diff
@@ -15,6 +15,8 @@
   $additionalparams,
 | -%>
 
+<% $comma = false %>
```
Comment (martinaamodt) [inline templates/cockroachdb.service.epp]: Why is this implemented here? Thought this was for another commit

File: manifests/init.pp [hunk 2]
```diff
@@ -38,6 +38,15 @@
 #   or the kernel log buffer.
 # [user]
 #   Sets the user/group that the processes are executed as.
+# [region]
```
Comment (martinaamodt) [inline manifests/init.pp]: Why are these comments in this commit?

**Review by martinaamodt — CHANGES_REQUESTED** [review]
See comments in code. Seems like you have code from some other unfinished features.
The task itself looks good

**Comment by Hexoplon** [comment]
> See comments in code. Seems like you have code from some other unfinished features.
> The task itself looks good

@martinaamodt Fixed. Removed old references from the code.

**Review by martinaamodt — APPROVED** [review]
Good:) Check out codefactor issues @Hexoplon (should be under ignored issues).

**Pull request merged**